gen-compdocs/comps/gen_kube_docs.go

```diff
@@ -55,9 +55,9 @@ func GenerateFiles(path, module string) error {
 		scheduler := schapp.NewSchedulerCommand()
 		return generators.GenMarkdownTree(scheduler, outDir, true)
 
-	case "kubelet":
-		kubelet := kubeletapp.NewKubeletCommand()
-		return generators.GenMarkdownTree(kubelet, outDir, true)
+	//	case "kubelet":
+	//	kubelet := kubeletapp.NewKubeletCommand()
```

Comment:
This should solve the compilation problem you're seeing:
```golang
kubelet := kubeletapp.NewKubeletCommand(context.TODO())
```
and let's get this back. After that is fixed I'll tag.

Reply:
@soltysh The generator has been updated. However, as you can see, the generated kubelet.md file doesn't contain any useful information. I am not quite sure if this is related to the change mtaufen merged several years ago (https://github.com/kubernetes/kubernetes/blame/master/cmd/kubelet/app/server.go#L172). After those refactoring, we had never been able to generate meaningful reference for kubelet. Do you have any ideas to restore this capability?

Reply:
That's my hope as well :sweat_smile: